recipe/meta.yaml

```
@@ -1,32 +1,32 @@
-{% set version = "3.3.1" %}
+{% set name = "secretstorage" %}
+{% set version = "3.4.0" %}
 
 package:
-  name: secretstorage
+  name: {{ name|lower }}
   version: {{ version }}
 
 source:
-  url: https://pypi.io/packages/source/S/SecretStorage/SecretStorage-{{ version }}.tar.gz
-  sha256: fd666c51a6bf200643495a04abb261f83229dcb6fd8472ec393df7ffc8b6f195
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: c46e216d6815aff8a8a18706a2fbfd8d53fcbb0dce99301881687a1b0289ef7c
 
 build:
-  number: 1
-  skip: true  # [win or py<36]
+  number: 0
+  skip: true  # [win or py<310]
   script: {{ PYTHON }} -m pip install . --no-deps --ignore-installed --no-cache-dir -vvv
 
 requirements:
   host:
-    - python
     - pip
-    - setuptools
-    - wheel
-    # needed to get pinning
-    - dbus 1.*
+    - python
+    - dbus
```

```suggestion
    - dbus {{ dbus }}
```
https://github.com/AnacondaRecipes/aggregate/blob/master/conda_build_config.yaml#L257